src/main/java/org/openhab/binding/zwave/handler/ZWaveSerialHandler.java

```diff
@@ -238,7 +240,7 @@ public void run() {
                             continue;
                         }
                     } catch (IOException e) {
-                        logger.error("Got I/O exception {} during receiving. exiting thread.", e.getLocalizedMessage());
+                        logger.debug("Got I/O exception {} during receiving. exiting thread.", e.getLocalizedMessage());
```

Comment:
Probably this should be WARN - this is quite a serious error.

Reply:
Changed to warn.